src/app/store.ts

```diff
@@ -1,6 +1,9 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import statement `combineSlices` is incorrect. Redux Toolkit uses `combineReducers` to combine multiple slice reducers. Please replace `combineSlices` with `combineReducers`.